src/main/java/core/basesyntax/strategy/impl/SupplyAmount.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.strategy.AmountHandler;
+
+public class SupplyAmount implements AmountHandler {
```

Comment:
```suggestion
public class SupplyAmountHandler implements AmountHandler {
```
same for other handlers